### ISSUE TITLE
inconsistent mat-snackbar screenreader behavior

### ISSUE THREAD

**Issue by diminutivesloop** [angular/material2#13116]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Screen readers should announce snack-bar messages.

#### What is the current behavior?
When using JAWS or NVDA with Chrome, if a mat-snackbar is invoked with a custom component and the `announcementMessage` option set, both the text content of the custom snack-bar and the `announcementMessage` string are announced to the user. However when using NVDA with other browsers such as IE, or Firefox only the `announcementMessage` string is announced. One way to work around this issue is to hide the custom snackbar from the screenreader via `aria-hidden` but this prevents the snackbar from being read to the user if they manually navigate to it.

#### What are the steps to reproduce?
https://stackblitz.com/edit/angular-gbkaav-pka2ed

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular: 6.1.7
Material: 6.4.7
NVDA: 2017.4
JAWS: 18.0.4104
Chrome: 68.0.3440.106
IE: 11.0.9600.18499
Firefox: 62.0
